docs/schema-overview.md

```diff
@@ -38,15 +39,17 @@ Pulsar schema resolves the pain points with the following capabilities:
 * keeps data compatibility on-track between schema versions. When new schemas are uploaded, the new versions can be read by old consumers.
 * stored in the existing storage layer BookKeeper, without additional system required.
 
-## How it works
+## How Pulsar Schema works
 
 Pulsar schemas are applied and enforced at the **topic** level. Producers and consumers can upload schemas to brokers, so Pulsar schemas work on both sides.
 
 ### Producer side
 
-This diagram illustrates how Pulsar schema works on the Producer side.
+This diagram illustrates how Pulsar schema works on the producer side.
 
-![How Pulsar schema works on the producer side](/assets/schema-producer.svg)
+![Workflow of Pulsar schema on the producer side](/assets/schema-producer.svg)
+
+Pulsar schema works on the Producer side in the following steps:
```

Comment:
```suggestion
Below are explanations for each step. 
```
We already have a featured snippet for this workflow. So keep this introductory sentence concise.

---

<img width="821" alt="image" src="https://github.com/apache/pulsar-site/assets/50226895/dab8c36d-0686-42c3-a92e-a75233038c6b">